cmd/jujud/agent/model/manifolds_test.go

```diff
@@ -166,3 +167,442 @@ func (s *ManifoldsSuite) TestClockWrapper(c *gc.C) {
 }
 
 type fakeClock struct{ clock.Clock }
+
+func (s *ManifoldsSuite) assertManifoldsDependencies(c *gc.C, manifolds dependency.Manifolds, expected map[string][]string) {
+	dependencies := make(map[string][]string, len(manifolds))
+	manifoldNames := set.NewStrings()
+
+	for name, manifold := range manifolds {
+		manifoldNames.Add(name)
+		dependencies[name] = []string{}
```

Comment:
I guess you need this assignment so you can explicitly say "this has no dependencies", but I think it would be better as nil rather than `[]string{}`.